tasks/gather_facts_ec2_security_groups.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+###
+### Gather ec2 security facts
+###
+
+- name: "({{ cluster.name }}) set ec2 security filter"
+  set_fact:
+    kops_ec2_security_groups_filters:
+      - key: vpc-id
```

Comment:
This would gather all security groups in this VPC. Imagine custom instances inside which might have their own security groups, independent of the cluster.

Is this desired? From the example in the Readme, I see you actually wanted to use security groups specified by name.

Reply:
The idea its to be able to just specify the name, the security group its actually created in the security-groups-rules.yml so because this is from k8s there shouldn't be any instances from another vpc(not even sure if its possible with kops) then it makes sense that its only within the same vpc

Reply:
@cytopia yes, it is intended like this. This is sub-optimal way, of course, but it let us to specify group name directly in cluster configuration.